pootle/apps/pootle_app/management/commands/calculate_checks.py

```diff
@@ -36,87 +33,16 @@ class Command(PootleCommand):
         ),
     )
     option_list = PootleCommand.option_list + shared_option_list
-
-    cached_methods = [CachedMethods.CHECKS]
     process_disabled_projects = True
 
     def handle_all_stores(self, translation_project, **options):
-        calculate_checks(check_names=options['check_names'],
-                         translation_project=translation_project)
+        QualityCheckUpdater(
+            options['check_names'],
+            translation_project=translation_project)
```

Comment:
I think you already noticed, but just for the record: Because if you just initialize the object, and the don't call any method, then no quality check is being calculated/updated/whatever.